src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let accumulator = initialValue;
+    let startIndex = 0;
+
+    if ([...arguments].length < 2) {
```

Comment:
why do you use spread here?
```suggestion
    if (arguments.length < 2) {
```